scripts/Jenkinsfile

```diff
@@ -1,47 +1,55 @@
 pipeline 
 {
 
-    agent 
+    agent
     {
         node 
         {
             label 'qeff_node'
         }
     }
-
     stages 
     {
         stage('Install QEfficient')
         {
             steps
-            {
-                sh '''
-                    python3.8 -m venv preflight_qeff
-                    . preflight_qeff/bin/activate
-                    pip install --upgrade pip setuptools
-                    pip install .[test] --extra-index-url https://download.pytorch.org/whl/cpu
-                    pip install junitparser pytest-xdist
-                    rm -rf QEfficient
-                '''
+            {                
+                script
+                {
+
+                    sh '''
+                    . ~/.bashrc
+                    docker run --privileged -dit --name ${BRANCH_NAME} -v ./:/efficient-transformers -v  ${HF_PATH}:${DOCKER_HF_PATH} ${DOCKER_LATEST}:master_latest
+                    docker exec ${BRANCH_NAME} bash -c "
+                    cd /efficient-transformers &&
+                    . /opt/qeff-env/bin/activate &&
+                    pip install --upgrade pip setuptools &&
+                    pip install .[test] --extra-index-url https://download.pytorch.org/whl/cpu &&
```

Comment:
Do we still need to have extra-index-url? we removed it in #139

Reply:
Updated to align with 139.